victkarangwa/WayFarer
# Bg travis status 167918433

#### What does this PR do?
- Change the connection string in .ENV file
#### Description of Task to be completed?
This PR changes the DB URL from `DB_URL` to `DATABASE_URL` to make postgrs reveal the connection string
#### How should this be manually tested?
- Navigate to https://adc-wayfarer-api.herokuapp.com 
- Use any endpoint
- You should be able to see the right response
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
